webapi/gtk3ui.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import unicode_literals
```

Comment:
There's little change in that compared with `gtkui.py`. Let's just rename and make changes to minimize diffs.

Reply:
Same changes as the .glade and .ui files (references to widget change to object). I'll make the suggested changes (rename and modify).